packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div class="comment">
```

Comment:
absolutely not on you but could u also add `kt-` prefix to this class, we r moving towards this in all our refactors, since this should be distinguished as a kotti class, and we will add this as a breaking change in the followup release, in case some user app relies on this